dist/dist-packages/linux/openziti-router/bootstrap.bash

```diff
@@ -31,6 +31,12 @@ makeConfig() {
           ZITI_ROUTER_PORT \
           ZITI_ROUTER_LISTENER_BIND_PORT="${ZITI_ROUTER_PORT}"
 
+  if [[ "${ZITI_ROUTER_ADVERTISED_ADDRESS}" =~ ^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
same comment here as the other PR. it's better to not try to detect this sort of thing and instead give people a way to specifically tell the system that it's using an IP address.